[2025-07-09T07:27:08.701+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T07:27:08.727+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T07:27:08.735+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T07:27:08.736+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T07:27:08.748+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): transform> on 2025-07-07 12:00:00+00:00
[2025-07-09T07:27:08.758+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6643) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T07:27:08.760+0000] {standard_task_runner.py:63} INFO - Started process 6657 to run task
[2025-07-09T07:27:08.760+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'transform', 'scheduled__2025-07-07T12:00:00+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpxw2epw7z', '--map-index', '4']
[2025-07-09T07:27:08.765+0000] {standard_task_runner.py:91} INFO - Job 580: Subtask transform
[2025-07-09T07:27:08.840+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [running]> on host e6a32b72ede0
[2025-07-09T07:27:08.942+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-07T12:00:00+00:00'
[2025-07-09T07:27:08.943+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T07:27:08.943+0000] {Untitled-1.py:139} INFO - Данные приняты из Xcom, начат процесс очистки
[2025-07-09T07:27:08.944+0000] {Untitled-1.py:71} INFO - Создание подключения к S3
[2025-07-09T07:27:08.944+0000] {Untitled-1.py:75} INFO - Подключение установлено, загрузка данных из S3
[2025-07-09T07:27:08.964+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T07:27:08.965+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T07:27:09.136+0000] {Untitled-1.py:81} INFO - Данные выгружены из S3
[2025-07-09T07:27:09.143+0000] {Untitled-1.py:148} INFO - Преобразование завершено, начинается передача DF в S3
[2025-07-09T07:27:09.145+0000] {Untitled-1.py:39} INFO - Создание подключения к S3
[2025-07-09T07:27:09.145+0000] {Untitled-1.py:44} INFO - Подключение установлено
[2025-07-09T07:27:09.154+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T07:27:09.155+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T07:27:09.592+0000] {Untitled-1.py:50} INFO - Загрузка данных в S3
[2025-07-09T07:27:09.763+0000] {Untitled-1.py:61} INFO - Загрузка завершена успешно
[2025-07-09T07:27:09.764+0000] {Untitled-1.py:156} INFO - Данные успешно сохранены в S3, передаю их в Xcom
[2025-07-09T07:27:09.764+0000] {Untitled-1.py:158} INFO - данные успешно очищены и записаны в хранилище
[2025-07-09T07:27:09.765+0000] {python.py:237} INFO - Done. Returned value was: {'path': {'bucket': 'airflare', 'key': 'temp/df_IKT_AER.json'}, 'rout': {'origin': 'IKT', 'destination': 'AER', 'bd': 'ikt_aer_outbound'}}
[2025-07-09T07:27:09.773+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T07:27:09.790+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=transform, run_id=scheduled__2025-07-07T12:00:00+00:00, map_index=4, execution_date=20250707T120000, start_date=20250709T072708, end_date=20250709T072709
[2025-07-09T07:27:09.820+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T07:27:09.842+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T07:27:09.844+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-09T11:24:58.389+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T11:24:58.428+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T11:24:58.448+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T11:24:58.449+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T11:24:58.471+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): transform> on 2025-07-07 12:00:00+00:00
[2025-07-09T11:24:58.490+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=21303) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T11:24:58.492+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'transform', 'scheduled__2025-07-07T12:00:00+00:00', '--job-id', '3277', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpb06fyzb1', '--map-index', '4']
[2025-07-09T11:24:58.494+0000] {standard_task_runner.py:63} INFO - Started process 21338 to run task
[2025-07-09T11:24:58.497+0000] {standard_task_runner.py:91} INFO - Job 3277: Subtask transform
[2025-07-09T11:24:58.605+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [running]> on host e6a32b72ede0
[2025-07-09T11:24:58.812+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-07T12:00:00+00:00'
[2025-07-09T11:24:58.815+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T11:24:58.820+0000] {Untitled-1.py:144} INFO - Извлечение данных из S3, подготовка к обработке
[2025-07-09T11:24:58.827+0000] {Untitled-1.py:70} INFO - Создание подключения к S3
[2025-07-09T11:24:58.845+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T11:24:58.848+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T11:24:59.578+0000] {Untitled-1.py:77} INFO - Подключение установлено
[2025-07-09T11:24:59.582+0000] {Untitled-1.py:81} INFO - Подключение установлено, загрузка данных из S3
[2025-07-09T11:24:59.877+0000] {Untitled-1.py:87} INFO - Данные выгружены из S3
[2025-07-09T11:24:59.891+0000] {Untitled-1.py:158} INFO - object
[2025-07-09T11:24:59.894+0000] {Untitled-1.py:159} INFO - object
[2025-07-09T11:24:59.895+0000] {Untitled-1.py:161} INFO - Преобразование завершено, начинается передача DF в S3
[2025-07-09T11:24:59.896+0000] {Untitled-1.py:38} INFO - Создание подключения к S3
[2025-07-09T11:24:59.909+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T11:24:59.910+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T11:25:00.194+0000] {Untitled-1.py:45} INFO - Подключение установлено
[2025-07-09T11:25:00.195+0000] {Untitled-1.py:49} INFO - Загрузка данных в S3
[2025-07-09T11:25:00.709+0000] {Untitled-1.py:60} INFO - Загрузка завершена успешно
[2025-07-09T11:25:00.710+0000] {Untitled-1.py:169} INFO - Данные успешно сохранены в S3
[2025-07-09T11:25:00.712+0000] {python.py:237} INFO - Done. Returned value was: {'path': {'bucket': 'airflare', 'key': 'temp/df_IKT_AER.json'}, 'rout': {'origin': 'IKT', 'destination': 'AER', 'bd': 'ikt_airflare_outbound'}}
[2025-07-09T11:25:00.713+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T11:25:00.753+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=transform, run_id=scheduled__2025-07-07T12:00:00+00:00, map_index=4, execution_date=20250707T120000, start_date=20250709T112458, end_date=20250709T112500
[2025-07-09T11:25:00.940+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T11:25:00.956+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-09T11:29:02.079+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T11:29:02.129+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T11:29:02.147+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [queued]>
[2025-07-09T11:29:02.149+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T11:29:02.172+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): transform> on 2025-07-07 12:00:00+00:00
[2025-07-09T11:29:02.197+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=22068) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T11:29:02.207+0000] {standard_task_runner.py:63} INFO - Started process 22102 to run task
[2025-07-09T11:29:02.206+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'transform', 'scheduled__2025-07-07T12:00:00+00:00', '--job-id', '3450', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpjfw884e9', '--map-index', '4']
[2025-07-09T11:29:02.208+0000] {standard_task_runner.py:91} INFO - Job 3450: Subtask transform
[2025-07-09T11:29:02.321+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.transform scheduled__2025-07-07T12:00:00+00:00 map_index=4 [running]> on host e6a32b72ede0
[2025-07-09T11:29:02.555+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-07T12:00:00+00:00'
[2025-07-09T11:29:02.560+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T11:29:02.564+0000] {Untitled-1.py:144} INFO - Извлечение данных из S3, подготовка к обработке
[2025-07-09T11:29:02.566+0000] {Untitled-1.py:70} INFO - Создание подключения к S3
[2025-07-09T11:29:02.592+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T11:29:02.596+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T11:29:03.290+0000] {Untitled-1.py:77} INFO - Подключение установлено
[2025-07-09T11:29:03.291+0000] {Untitled-1.py:81} INFO - Подключение установлено, загрузка данных из S3
[2025-07-09T11:29:03.528+0000] {Untitled-1.py:87} INFO - Данные выгружены из S3
[2025-07-09T11:29:03.542+0000] {Untitled-1.py:158} INFO - object
[2025-07-09T11:29:03.543+0000] {Untitled-1.py:159} INFO - object
[2025-07-09T11:29:03.544+0000] {Untitled-1.py:161} INFO - Преобразование завершено, начинается передача DF в S3
[2025-07-09T11:29:03.545+0000] {Untitled-1.py:38} INFO - Создание подключения к S3
[2025-07-09T11:29:03.560+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T11:29:03.562+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T11:29:03.778+0000] {Untitled-1.py:45} INFO - Подключение установлено
[2025-07-09T11:29:03.779+0000] {Untitled-1.py:49} INFO - Загрузка данных в S3
[2025-07-09T11:29:04.212+0000] {Untitled-1.py:60} INFO - Загрузка завершена успешно
[2025-07-09T11:29:04.214+0000] {Untitled-1.py:169} INFO - Данные успешно сохранены в S3
[2025-07-09T11:29:04.216+0000] {python.py:237} INFO - Done. Returned value was: {'path': {'bucket': 'airflare', 'key': 'temp/df_IKT_AER.json'}, 'rout': {'origin': 'IKT', 'destination': 'AER', 'bd': 'ikt_airflare_outbound'}}
[2025-07-09T11:29:04.217+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T11:29:04.243+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=transform, run_id=scheduled__2025-07-07T12:00:00+00:00, map_index=4, execution_date=20250707T120000, start_date=20250709T112902, end_date=20250709T112904
[2025-07-09T11:29:04.302+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T11:29:04.349+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T11:29:04.352+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
